README.md

```diff
@@ -26,21 +26,74 @@ following rules, which we consider useful.
 
 ## Belt Activities
 
-The belt activities serve as a checklist for the team and give additional information to accomplish these activitites. In particular, they consist of:
-- a brief task description the team,
+The belt activities serve as a checklist for the team and give additional information to accomplish these activities. In particular, they consist of:
+- a brief task description for the team,
 - a list of further readings to learn more about this topic,
-- references to related activities (if necessarry),
+- references to related activities (if necessary),
 - the benefits of this activity, and
-- a checklist for the asssessment of the belt.
+- a checklist for the assessment of the belt.
 
-## More Headings
+## Roles
 
-TODO: Assessment and other topics
+Security Belts depends on the existence of several roles from the Agile methodology such as developer and product owner. Developers, product owner, and optionally a scrum master, form the so-called *team*. 
+
+### Security Champion
+
+The maturity model Security Belts adds the role of the Security Champion to the Agile methodology. The Security Champion is a developer, who is also part of the team. He/she has a profound security knowledge but is not necessarily a security expert. The Security Champion empowers the team such that they can ensure the security of their software products by themselves. Moreover, he/she enables the team to improve themselves concerning software security and raises the importance of security within the team (developers and product owner) as well as towards the management.
```

Comment:
I don't like the he/she, maybe we can just use bullet points and thus do not need to start each sentence with "The security champion"?
```suggestion
The maturity model Security Belts adds the role of the Security Champion to the Agile methodology. The Security Champion ...
  - is a developer, who is also part of the team.
  - has profound security knowledge but is not necessarily a security expert.
  - empowers the team such that they can ensure the security of their software products by themselves.
  - enables the team to improve themselves concerning software security and raises the importance of security within the team (developers and product owner) as well as towards the management.
```
Maybe the next point about the training could be added as well